packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -138,15 +135,10 @@ export class FluidDevtools implements IFluidDevtools {
 
 	/**
 	 * Stores Container-level devtools instances registered with this object.
-	 * Maps from a {@link ContainerKey} to the corresponding {@link ContainerDevtools} instance.
+	 * Maps from a {@link ContainerKey} to the corresponding {@link BaseDevtools} instance.
 	 */
```

Comment:
Let's do separate comments for each property. Note that a TSDoc comment specifically applies to the thing that follows it. So in order for Intellisense, etc. to get the right behavior, we need a separate comment for each.